### ISSUE TITLE
Minimizing disk requirements for Travis.

### ISSUE THREAD

**Issue by TheChymera** [IBT-FMI/gebuilder#25]
Now that we're able to [successfully build larger packages](https://github.com/IBT-FMI/gebuilder/issues/24), it seems we can also easily (surpass TravisCI's capabilities)[https://travis-ci.org/IBT-FMI/SAMRI/jobs/400095225#L437].

I'm wondering whether there's anything more we can do to minimize the size of resulting docker images. Cleaning distfiles would definitely be one thing, but there's other things as well. I see we're already shipping a [`cleanportage.sh` file](https://github.com/IBT-FMI/gebuilder/blob/master/gebuilder/scripts/common/cleanportage.sh) which right now does nothing.

Right now and for as long as we're not struggling with significant USE flag updated and network bottlenecks at build time, I think it would be most sensible to always wipe everything that can be wiped for the specialization case. Where would you say this is best done, @Doeme ?

**Comment by Doeme** [comment]
Probably in `gebuilder/scripts/docker_image/default/99-cleanup.sh`. I'm not sure what you want to clean up, though. The portage-tree should not be inside the image, as it is bind-mounted. Theoretically one could remove the build-time only dependencies (autoconf, ...), but I'm not sure on how to figure those out...

**Comment by TheChymera** [comment]
I meant more along the lines of [this](https://wiki.gentoo.org/wiki/Knowledge_Base:Freeing_disk_space). I looked inside the image and it looks like `/usr/portage/distfiles/*` aren't deleted.

Also, I was thinking of cleaning up disk space more generally? like after each specialization update? I'm guessing we want all images to be as small as possible, not just the Docker ones. What would be the best place for that? Here, [between 30 and 99](https://github.com/IBT-FMI/gebuilder/tree/master/gebuilder/scripts/update/default)?

**Comment by Doeme** [comment]
Yeah, seems reasonable. But: /usr/portage/ _should_ be empty anyways, because they get bind-mounted from the stemgentoo, s.t. we do not need to download every package and the portage-tree NUM_IMAGES times.
If it isn't, that's a bug.
